bootstrapvz/providers/gce/tasks/image.py

```diff
@@ -56,6 +26,26 @@ def run(cls, info):
 		if 'description' in info.manifest.image:
 			image_description = info.manifest.image['description']
 		log_check_call(['gcutil', '--project=' + info.manifest.image['gce_project'],
-		                'addimage', info._gce['image_name'],
-		                info.manifest.image['gcs_destination'] + info._gce['tarball_name'],
+		                'addimage', info.image['name'],
+		                info.manifest.image['gcs_destination'] + info.image['tarball_name'],
```

Comment:
This step is optional: the tarball is only uploaded [if the user wants to][1]. Also, its name isn't really defined by the user, but by [this definition][2], which was made (or at least commited) by @rybaktomasz.

[1]: https://github.com/andsens/bootstrap-vz/blob/0068cdc1a8a15061856a962045984126476b9583/bootstrapvz/providers/gce/__init__.py#L62
[2]: https://github.com/andsens/bootstrap-vz/blob/0068cdc1a8a15061856a962045984126476b9583/bootstrapvz/providers/gce/tasks/image.py#L8